maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/SubTreeRouteHandler.swift

```diff
@@ -53,9 +56,38 @@ final class SubTreeRouteHandler: HTTPHandler {
     
     private func getErrorCode(message: String) -> String {
         if message.contains("Error kAXErrorIllegalArgument getting snapshot for element") {
-           return "illegal-argument-snapshot-failure"
+            return "illegal-argument-snapshot-failure"
         } else {
-           return "unknown-snapshot-failure"
+            return "unknown-snapshot-failure"
+        }
+    }
+    
+    private func tapOnSystemPermissionAlertIfNeeded(springboardApplication: XCUIApplication, appName: String) {
+        let predicate = NSPredicate(format: "label CONTAINS[c] %@", appName)
+        
+        guard let data = UserDefaults.standard.object(forKey: "permissions") as? Data,
+              let permissions = try? JSONDecoder().decode([String : PermissionValue].self, from: data),
+              let notificationsPermission = permissions.first(where: { $0.key == "notifications" }) else {
+            return
```

Comment:
Another question would be: "notifications" would work in case of localization?

Reply:
This string comes from the flow definition. Flow definitions currently do not support localisation.